### ISSUE TITLE
[BUG] Dot expansion code expands file paths

### ISSUE THREAD

**Issue by ipspace** [ipspace/netlab#774]
### Describe the bug

The dot expansion code should have avoided key values that contain '/' to prevent expanding file paths. Looks like that part of the code is broken.

### To Reproduce

Create the provider configuration files with **netlab create** and inspect the mess generated in clab.yml

### Expected behavior

**binds** list should contain valid filenames not JSON values

### Lab topology

```
provider: clab

nodes:
  h:
    device: linux
    clab.binds:
      ./dhcp.cfg: /data/dhcp.cfg
```

### Version

* netlab version 1.5.1

**Comment by ipspace** [comment]
This is worse than I thought; the dot expansion happens within newer versions of Box. We have to undot the values.

**Comment by ipspace** [comment]
Implemented in #775
